### PR TITLE
feat(j-s): Court of Appeal ruling screen

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# Court of Appeal ruling screen

[Landsréttur - úrskurðarskjár hjá Landsrétti
](https://app.asana.com/0/1199153462262248/1204298513132592)

## What
Added new screen for court of appeal case ruling
Added new data columns to the Case table appeal ruling decision and appeal conclusion 

## Why

So that the cour of appeal can register ruling on case 

## Screenshots / Gifs
Screenshot 2023-04-27 at 16 17 28

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/src/app/modules/case/dto/updateCase.dto.ts [hunk 1]
```diff
@@ -369,6 +370,16 @@ export class UpdateCaseDto {
...
 
+  @IsOptional()
+  @IsString()
+  @ApiPropertyOptional()
+  readonly appealConclusion?: string
+
+  @IsOptional()
+  @IsEnum(CaseAppealRulingDecision)
+  @ApiPropertyOptional({ enum: CaseAppealRulingDecision })
+  readonly appealRulingDecision?: CaseAppealRulingDecision
```
[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/backend/src/app/modules/case/dto/updateCase.dto.ts:381]: In addition, these fields should be added to the type `UpdateCase` in `limitedAccessCase.service.ts`.

Please consider adding also adding the following fields that are missing from this type:
- `appealCaseNumber`
- `appealAssistantId`
- `appealJudge1Id`
- `appealJudge2Id`
- `appealJudge3Id`

File: libs/judicial-system/types/src/lib/case.ts [hunk 2]
```diff
@@ -270,6 +279,8 @@ export interface Case {
   appealJudge2?: User
   appealJudge3?: User
   appealReceivedByCourtDate?: string
+  appealConclusion?: string
+  appealRulingDecision?: CaseAppealRulingDecision
```
Comment (gudjong) [inline libs/judicial-system/types/src/lib/case.ts:283]: Please also add these fields to the type `UpdateCase` below.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/components/FormProvider/caseGql.ts [hunk 3]
```diff
@@ -199,6 +199,8 @@ const CaseQuery = gql`
       prosecutorStatementDate
       defenderStatementDate
       appealReceivedByCourtDate
+      appealConclusion
```
[the remark this answers is not in the document]
Comment (MargretFinnboga) [reply apps/judicial-system/web/src/components/FormProvider/caseGql.ts:202]: I cant seem to find `limitedAccessUpdateCaseGql.ts` 🤔

[the remark this answers is not in the document]
Comment (gudjong) [reply apps/judicial-system/web/src/components/FormProvider/caseGql.ts:202]: Could be a new thing that is not merged yet

**Review by gudjong — APPROVED** [review]

**Pull request merged**